covid_model/disconnected_to_connected_betas_co_regions.py

```diff
@@ -0,0 +1,260 @@
+import json
```

Comment:
This script does all the heavy lifting for adjusting betas from disconnected betas to connected betas. Produces a bunch of plots in the output directory.

Reply:
Good idea, will put this in the refactoring TODO card.